compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -375,97 +371,168 @@ setReductionVectorDistributionConfig(IREE::GPU::TargetAttr target,
       foundSingleReductionOutput = true;
       continue;
     }
-    if (!op.getMatchingIndexingMap(&initOpOperand).isIdentity()) {
+    if (!op.getMatchingIndexingMap(&initOpOperand).isIdentity())
       return failure();
-    }
   }
   if (!foundSingleReductionOutput) {
     return failure();
   }
 
+  return success();
+}
+
+static llvm::FailureOr<IREE::GPU::LoweringConfigAttr>
+getVectorDistributeReductionConfig(linalg::LinalgOp op,
+                                   IREE::GPU::TargetAttr target,
+                                   ArrayRef<unsigned> sharedWgpDims,
+                                   int64_t workgroupSize, int64_t subgroupSize,
+                                   int64_t threadLoads) {
+  auto context = op.getContext();
```

Comment:
Since the type is not immediately obvious based on the rhs only:

```suggestion
  MLIRContext* context = op.getContext();
```